### ISSUE TITLE
Unable to turn DNS filtering on when using Voice Over screen reader

### ISSUE THREAD

**Issue by evgeniy-ADG** [AdguardTeam/AdguardForiOS#170]
Adguard settings window will close if you try toggling DNS status.

**1423327**
